### DISCUSSION TITLE
Tenants enhancement: In the case of too many tenants, whether to support the classification of the tenant list

### DISCUSSION THREAD

**Discussion by pgy866** [OrchardCMS/OrchardCore#10397] [Ideas]
Tenants enhancement: In the case of too many tenants, whether to support the classification of the tenant list

**Comment by Piedone** [comment]
Do you mean like having tenant categories? I think this is a great idea. Could you open a feature request [here](https://github.com/OrchardCMS/OrchardCore/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=)? The issue tracker is where we can better see feature requests.

**Reply by hishamco** [reply]
#10402
